client/trademon/src/components/tileComponents/myProfileTileComponents/profileBannerComponent.tsx

```diff
@@ -16,19 +16,13 @@ export default function ProfileBanner() {
   }, []);
 
   const setSellerRating = () => {
-    let totalRating: number = 0;
-    userData.sellerRating.forEach((rating: number) => {
-      totalRating += rating;
-    });
-    setSellerRatingValue(totalRating/userData.sellerRating.length);    
+    let totalRating: number = calcRating(userData.sellerRating);
```

Comment:
Neat. Helper function making 5 lines into 2

Reply:
Its cause I use that same function in a few places on different files (: